### DISCUSSION TITLE
[Concept] Dynamite

### DISCUSSION THREAD

**Discussion by GhostsModels** [EtherealEmil/UsualScrap#36] [Ideas]
60-80 cost item that is lit and after a certain amount of time blows up dealing massive amounts of damage to enemies. Basically a retextured landmine that is purposefully triggerable. Probably should be a placable for balance reasons, but maybe make it throwable. I'd be cool if it was a oneshot (like just make the damage number nuts) then with different mods that make all enemies killable, you could set them super high as to make this the only practical way to kill them.

**Comment by EtherealEmil** [comment]
I've also had idea for a handheld explosive that can be thrown but just haven't acted on it yet. Depending on how it functions I would make it either a store or scrap item. A explosive that you can buy and can kill mostly anything would be strong unless it's expensive or has an inconsistent fuse time.
